playground/hmr/__tests__/hmr.spec.ts

```diff
@@ -786,4 +786,14 @@ if (import.meta.hot) {
     )
     await untilUpdated(() => el.textContent(), '2')
   })
+
+  test('issue-3303', async () => {
```

Comment:
Hi,
I just noticed that the issue number doesn't match with the issue in the description.
There are multiple places where it should change if I am not mistaken 😅 .
```suggestion
  test('issue-3033', async () => {
```

Reply:
Great catch! I've changed.